enrysrv/cli/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"os"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	appname = "enrysrv"
+)
+
+var (
+	version = "undefined"
+	build   = "undefined"
+	commit  = "undefined"
```

Comment:
I think it's better just link this variables during building time.

Eventually if this project it's integrated with src-d/ci, link this variables trough building flags it's the way [Makefile's CI](https://github.com/src-d/ci/blob/master/Makefile.main#L127) do it.